CHANGELOG.md

```diff
@@ -15,36 +17,62 @@ rows in this table:
 SELECT table_rows FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'cromwell' AND table_name = 'JOB_STORE_SIMPLETON_ENTRY';
 ```
 
-### Disable call-caching for tasks
+#### Execution Directory Layout (cache copies)
+
+Attempts to copy call cached results will no longer occupy the same directory as the first attempt to run the task, should the cache copy fail. 
+Instead they will get their own subdirectory of the call root, eg:
+
+```
+[...]/callRoot/
```

Comment:
[ToL]
Might a "before" be helpful in addition to the "after"? This was tricky to figure out and I say it doesn't hurt to write down what's in our heads.